src/OrleansCodeGenerator/Boostrap.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace OrleansCodeGenerator
+{
+    using Orleans.CodeGeneration;
+
+    public class Boostrap : IRuntimeCodeGenerator
```

Comment:
I completely agree, this class should have been renamed as its purpose evolved. Will fix.